hack/fix-csr-signer.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
```

Comment:
Would be good to have a comment header explaining why this scripts exists.

Reply:
I'm having issues with the workaround. Since it's not critical for the release, I removed it from this PR.